### ISSUE TITLE
Skipping a song from spotify during playback breaks next song queuing

### ISSUE THREAD

**Issue by EddieTheCubeHead** [EddieTheCubeHead/Stagnum#213]
Depends on #183

Reproduction:

- Create a pool
- Skip a song **from spotify**
- Observe console logs when Stagnum tries to queue next song

Expected result:

- Playback state should be fetched, queue event should be deferred accordingly, queuing should eventually succeed near song end

Actual result:

- Error when fetching playback data from database

**Comment by EddieTheCubeHead** [comment]
As #183 was not yet merged to master, the bugfix was added directly into #183 and will be merged as a part of it.
